mediagrains/numpy/convert.py

```diff
@@ -152,11 +152,12 @@ def _convert_rgb_to_yuv444(grain_in: VideoGrain, grain_out: VideoGrain) -> None:
                  grain_in.component_data.G,
                  grain_in.component_data.B)
 
-    np.clip((R*0.2126 + G*0.7152 + B*0.0722), 0, 1 << bd, out=grain_out.component_data.Y)
-    np.clip((R*-0.114572 - G*0.385428 + B*0.5 + (1 << (bd - 1))),
-            0, 1 << bd, out=grain_out.component_data.U)
-    np.clip((R*0.5 - G*0.454153 - B*0.045847 + (1 << (bd - 1))),
-            0, 1 << bd, out=grain_out.component_data.V)
+    np.clip((R*0.2126 + G*0.7152 + B*0.0722),
```

Comment:
Nit: worth an explanatory comment? Although it is in the blame I suppose
```suggestion
    # Note that NumPy requires the `unsafe` cast here due to the slight loss of precision in RGB<->YUV conversion
    np.clip((R*0.2126 + G*0.7152 + B*0.0722),
```